app/src/main/java/com/example/cooky/data/remote/api/ApiConstant.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.cooky.data.remote.api
+
+const val BASE_URL = "https://api.spoonacular.com/"
+const val PATH_RECIPE = "recipes"
+const val PATH_FOOD = "food"
+const val PATH_INGREDIENTS = "ingredients"
+const val PATH_SEARCH = "search"
+const val PATH_RANDOM = "random"
+const val PATH_INFORMATION = "{id}/information"
+const val PATH_AUTOCOMPLETE = "autocomplete"
+const val PATH_NUTRITION = "{id}/nutritionWidget.json"
+const val PATH_FINDBYNUTRIENTS = "findByNutrients"
+const val PATH_COMPLEXSEARCH = "complexSearch"
+
+const val QUERY_QUERY = "query"
+const val QUERY_CUISINE = "cuisine"
+const val QUERY_DIET = "diet"
+const val QUERY_INCLUDE_INGREDIENTS = "includeIngredients"
+const val QUERY_EXCLUDE_INGREDIENTS = "excludeIngredients"
+const val QUERY_INTOLERANCES = "intolerances"
+const val QUERY_NUMBER = "number"
+const val QUERY_TAG = "tags"
+const val QUERY_ID = "id"
+const val QUERY_MAX_READYTIME = "maxReadyTime"
+const val QUERY_SORT = "sort"
+const val QUERY_SORTDIRECTION = "sortDirection"
+const val QUERY_MIN_CARB = "minCarbs"
+const val QUERY_MAX_CARB = "maxCarbs"
+const val QUERY_MIN_PROTEIN = "minProtein"
+const val QUERY_MAX_PROTEIN = "maxProtein"
+const val QUERY_MIN_CALORIES = "minCalories"
+const val QUERY_MAX_CALORIES = "maxCalories"
+const val QUERY_MIN_FAT = "minFAT"
+const val QUERY_MAX_FAT = "maxFAT"
+const val QUERY_MIN_CALCIUM = "minCalcium"
+const val QUERY_MAX_CALCIUM = "maxCalcium"
+const val QUERY_MIN_CHOLESTEROL = "minCholesterol"
+const val QUERY_MAX_CHOLESTEROL = "maxCholesterol"
+const val QUERY_MIN_SUGAR = "minSugar"
+const val QUERY_MAX_SUGAR = "maxSugar"
+const val QUERY_MIN_VITAMIN_A = "minVitaminA"
```

Comment:
em suggest những thuộc tính consttant giống nhau em có thể cho ra 1 class khác nhé. 
Thằng Constant này các lúc càng phình to.
vừa lại clear hơn